src/index.ts

```diff
@@ -74,6 +98,23 @@ export function normalizeLicenseText(
   return normalizedLines.join('\n').trim();
 }
 
+function stripControlCharacters(
+  line: string,
+  characters: ControlCharacter[],
+): string {
+  const controlCharPositions: number[] = [];
+  for (let charIndex = 0; charIndex < line.length; charIndex++) {
+    if (characters.includes(line.charAt(charIndex))) {
```

Comment:
It might be slightly more readable to give `characters` a name that indicates that these are the characters that we want to strip out, but I don't have a strong opinion on that either way.